### PR TITLE
DOC: add plugin description

### LINKED ISSUE #36: add `description` and `short_description` to `Plugin` instantiation
q2-feature-table has [examples](https://github.com/qiime2/q2-feature-table/blob/master/q2_feature_table/plugin_setup.py#L28) of these, and we're filling them in for the other plugins now.

### PR DESCRIPTION
**Pull request by gregcaporaso**
fixes #36

### CODE REVIEW

File: q2_deblur/plugin_setup.py [hunk 1]
```diff
@@ -24,13 +24,15 @@
...
     citation_text=("Deblur rapidly resolves single-nucleotide community "
                    "sequence patterns. Amnon Amir, Daniel McDonald, Jose "
                    "A. Navas-Molina, Evguenia Kopylova, Jamie Morton, "
                    "Zhenjiang Zech Xu, Eric P. Kightley, Luke R. Thompson, "
                    "Embriette R. Hyde, Antonio Gonzalez, Rob Knight. mSystems "
-                   "Mar 2017, 2 (2) doi:10.1128/mSystems.00191-16.")
+                   "Mar 2017, 2 (2) doi:10.1128/mSystems.00191-16."),
+    description=('This QIIME 2 plugin wraps the Deblur software for '
+                 'performing sequence quality control.'),
+    short_description='Plugin for sequence quality control with Deblur.'
```
Comment (wasade) [inline q2_deblur/plugin_setup.py]: Is this quality control, or sequence variant assessment?

Comment (ebolyen) [inline q2_deblur/plugin_setup.py]: We're using similar language for `dada2`. I think this wording is kind of making the assumption that you'd naturally have sequence variants after proper quality-control/derep.

Comment (gregcaporaso) [inline q2_deblur/plugin_setup.py]: @wasade, I'm fine with whatever changes you want to make. You can close this PR and open a new one if you have different language that you'd like to use. Just let us know if you're going to do that ASAP so we can get this in.

Comment (wasade) [inline q2_deblur/plugin_setup.py]: I'm in favor of consistency. If this is the language used for dada2 then lets proceed with it

**Pull request merged**